.husky/pre-commit

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+# Section for git-secrets
+if ! command -v git-secrets &> /dev/null
+then
+    echo "git-secrets is not installed. Please run 'brew install git-secrets' or visit https://github.com/awslabs/git-secrets#installing-git-secrets"
+    exit 1
+fi
+
+# Initialise git-secrets configuration
+git-secrets --register-aws > /dev/null
+
+echo "Running git-secrets..."
+# Scans all files that are about to be committed.
+git-secrets --pre_commit_hook -- "$@"
+
```

Comment:
You can add the command here and it will run for every commit after it is stagged
```suggestion
npm run secretScan
```

Reply:
This is redundant.